Comp2022/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -430,7 +431,8 @@ void Drivetrain::ResetGyro()
 
 degree_t Drivetrain::GetHeadingAngle()
 {
-    return (m_pigeonValid) ? (-m_gyro.GetFusedHeading() * 1_deg) : 0_deg;
+    //return (m_pigeonValid) ? (-m_gyro.GetFusedHeading() * 1_deg) : 0_deg;

```

Comment:
Just curious, why did you comment out the pigeon valid check line? Could you have just removed the negative to invert it?

Reply:
We were having some problems making the pigeon be seen as valid in simulation, so commenting that out was just a temporary test. I’ll uncomment that soon